commands/mpool.go

```diff
@@ -47,13 +50,58 @@ var mpoolLsCmd = &cmds.Command{
 				if err != nil {
 					return err
 				}
-				fmt.Fprintln(w, c.String()) // nolint: errcheck
+				_ = PrintString(w, c)
 			}
 			return nil
 		}),
 	},
 }
 
+var mpoolShowCmd = &cmds.Command{
+	Helptext: cmdkit.HelpText{
+		Tagline: "Show content of an outstanding message",
+	},
+	Arguments: []cmdkit.Argument{
+		cmdkit.StringArg("cid", true, false, "The CID of the message to delete"),
```

Comment:
BLOCKING: argument message looks like copypasta from the delete command

Reply:
Eek, thanks